### ISSUE TITLE
Allow connections to Mongo using "mongodb+srv"

### ISSUE THREAD

**Issue by NathanielBeen** [man-group/ArcticDB#1512]
**Is your feature request related to a problem? Please describe.**
Our Mongo database is configured to use srv connection strings (https://www.mongodb.com/developer/products/mongodb/srv-connection-strings/) but attempting to connect to this database through Arctic fails with a 'ValueError: Invalid URI specified' error.

**Describe the solution you'd like**
This issue is caused by the `MongoLibraryAdapter.supports_uri` method. Adjusting the method to
```@staticmethod
    def supports_uri(uri: str) -> bool:
        return uri.startswith("mongodb://") or uri.startswith("mongodb+srv://")
```
 allows us to connect to Arctic without issue.

**Comment by alexowens90** [comment]
Hi @NathanielBeen, this is fixed in #1513, and will be available in the next release of ArcticDB

**Comment by NathanielBeen** [comment]
Great thank you!
